src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -226,7 +226,7 @@ that this verification is disruptive.
 
 #### `benchmark throughput`
 
-`subctl benchmark throughput <kubeconfig1> [<kubeconfig2>] [flags]`
+`subctl benchmark throughput --kubecontexts <context1>[,<context2>] [flags]`
```

Comment:
The command requires two kubecontexts.

```suggestion
`subctl benchmark throughput --kubecontexts <context1>,<context2> [flags]`
```

Reply:
when running with `--intra-cluster` then only one kubecontext is needed